src/components/DefineRegistration/StartDate.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <v-card flat class="rounded-4 py-1">
+    <div class="section-container step-container">
+      <v-row no-gutters>
+        <v-col cols="12" md="2" lg="2">
+          <label>Start Date</label>
+        </v-col>
+        <v-col cols="12" md="10" lg="10" id="start-date-selector" class="pl-8">
+          <date-picker
+            ref="startDateRef"
+            title="Start Date"
+            :nudgeRight="40"
+            :nudgeTop="85"
+            :initialValue="null"
+            :minDate="startDateMinStr"
+            :maxDate="startDateMaxStr"
+            :inputRules="startDateRules"
+            @emitDateSync="setRegistrationStartDate($event)"
+          />
+        </v-col>
+      </v-row>
+    </div>
+  </v-card>
+</template>
+
+<script lang="ts">
+// Libraries
+import { Component, Mixins } from 'vue-property-decorator'
+import { Action, Getter } from 'vuex-class'
+import { ActionBindingIF } from '@/interfaces'
+
+// Shared Components
+import { DatePicker } from '@bcrs-shared-components/date-picker'
+import { RuleHelpers } from '@/rules'
+import { DateMixin } from '@/mixins'
+
+@Component({
+  components: {
+    DatePicker
+  }
+})
+export default class StartDate extends Mixins(DateMixin) {
+  // Global actions
+  @Action setRegistrationStartDate!: ActionBindingIF
+
+  /** The minimum start date that can be entered (Up to 2 years ago today). */
+  private get startDateMin (): Date {
+    const startDateMin = new Date()
```

Comment:
Use date-mixin getter for "today" (or "now"). (Also everywhere you use `new Date()`.)

This is so that we don't need to trust the user's system clock.

Reply:
`@Getter getCurrentJsDate!: Date`

Reply:
Sounds good, good call.

Reply:
To recap:
- a Date object is stored as UTC internally
- creating a Date object converts from the user's timezone (which might not be Pacific) to internal UTC
- most Date output methods convert from internal UTC to the user's timezone (which might not be Pacific)
- instead, use the "toLocale" versions of the date methods to convert to locale "en-US" and timezone "America/Vancouver" -- the necessary helper methods probably already exist in date-mixin.ts !

Reply:
Updated!